source/GlassView.Export/Configuration/Extensions.cs

```diff
@@ -0,0 +1,24 @@
+using System.Text.Json;
+using Microsoft.Extensions.Configuration;
+
+using static Atmoos.GlassView.Export.Extensions;
+
+namespace Atmoos.GlassView.Export.Configuration;
+
+internal static class Extensions
+{
+    public static JsonSerializerOptions Configure(this JsonSerializerOptions options, JsonFormatting? formatting)
```

Comment:
I feel like we have JsonSerializerOptions in too many places: Export, Core & Tests. We should keep only one place where we configure them and then reference that place.
For example in the tests we could use the json options from Core since we anyway reference the Core assembly in the test projects.

Reply:
In  principle I agree with you. But I'd argue that place already exists and is [here](https://github.com/atmoos/GlassView/blob/exportBenchmarkResults/source/GlassView.Core/Extensions.cs) and we use it just like you're proposing we should.

This method "only" maps` JsonFormatting` options, the ones that we (i.e. the user)  configure in the app settings, onto the 'JsonSerializerOptions' that we pass as parameter to this method. Perhaps I should rename this internal method to `Map`?

But I think this is something we better discuss directly...

Reply:
From a readability point of view, it is fine as it is.